packages/laravel/config/hybridly.php

```diff
@@ -42,6 +42,8 @@
         'application_main' => Architecture::APPLICATION_MAIN,
         'root_view' => Architecture::ROOT_VIEW,
         'extensions' => ['vue', 'tsx'],
+        // Don't load vue files as views if they are under the following directories
+        'excluded_views_directories' => ['Components'],
```

Comment:
Would you mind changing this to lowercase to keep consistency with the existing conventions?

Reply:
In this case we don't need anything by default, because any directory with the name of the component or layout directory will already be excluded, this is just a default so it can handle both uppercase and lowercase